docs-ref-conceptual/azure-cli-vm-tutorial.yml

```diff
@@ -212,7 +212,7 @@ items:
         variable right away, since it's returned in the VM creation information. If you'd need other information about the VM later, it can always be
         obtained from the `az vm show` command.
 
-        ```bash
+        ```azurecli
```

Comment:
Why not use azurecli-interactive here (if we're allowed to use it)?

Reply:
This one was in a different section and by itself.